ecl/hql/hqldesc.cpp

```diff
@@ -190,6 +296,29 @@ void expandScopeSymbolsMeta(IPropertyTree * meta, IHqlScope * scope)
     scope->getSymbols(symbols);
     symbols.sort(compareSymbolsByPosition);
 
-    ForEachItemIn(i, symbols)
-        expandSymbolMeta(meta, &symbols.item(i));
-}
+    IHqlExpression * expr = queryExpression(scope);
+    HqlExprArray baseSymbols;
+
+    ForEachChild(i, expr){
+        IHqlExpression * cur = expr->queryChild(i);
+        IHqlScope * curBase = cur->queryScope();
+        if (curBase) {
+            curBase->getSymbols(baseSymbols);
+        }
+    }
+
+
+    ForEachItemIn(i, symbols){
```

Comment:
Are you talking about this function - https://github.com/hpcc-systems/HPCC-Platform/blob/master/ecl/hql/hqlexpr.hpp#L1012 . Actually, I tried to use it initially, but it takes a lookupcontext parameter. How would I go about getting it ?